packages/wallet/src/services/AssetsTracker.ts

```diff
@@ -47,16 +51,17 @@ export const createAssetsTracker = (
         // Fetch asset metadata only for assets not already present in assetsMap
         map((assetIds) =>
           assetIds.map((assetId) => {
-            if (assetsMap.has(assetId)) {
-              return of(assetsMap.get(assetId));
+            const assetInfo = assetsMap.get(assetId);
+            if (assetInfo && isAssetInfoComplete(assetInfo)) {
+              return of(assetInfo);
             }
-            logger.debug(`Fetching metadata for asset ${assetId}`);
+            logger.debug('Fetching data data for ', assetId);
```

Comment:
```suggestion
            logger.debug('Fetching asset data for', assetId);
```

Reply:
done in [147b1d0](https://github.com/input-output-hk/cardano-js-sdk/pull/464/commits/147b1d0dbf1a900cb6426178b96e45b265cddd42)